modules/godot/systems/timer_updater_system.cpp

```diff
@@ -0,0 +1,30 @@
+#include "timer_updater_system.h"
+#include "core/os/os.h"
+#include "scene/main/scene_tree.h"
+
+void timer_updater_system(TimersDatabag *td,
+		SceneTreeInfoDatabag *p_sti,
```

Comment:
Please add `const` to take this DB immutable, so it doesn't interfere with other systems eventually.